internals/daemon/response.go

```diff
@@ -169,22 +150,31 @@ func (f fileResponse) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	http.ServeFile(w, r, string(f))
 }
 
+// Reponsef behaves similar to the other printf style functions as long as additional
+// arguments beyond the format string are provided.
+//
+// Note: If no arguments are provided, formatting is disabled, and the format string
+// is used as is and not interpreted in any way.
+func Reponsef(status int, format string, v ...interface{}) Response {
```

Comment:
Note there's a typo: `Reponsef` -> `Responsef`.

However, this doesn't just return a formatted response, it's an *error* response. I think we need to convey that. Maybe `Errorf` or `ErrorResponsef` (the shorter version is nice, but might be too confusing with `fmt.Errorf`).

Alternatively, we could keep it `Responsef` but make `status==http.StatusOK` do something sensible. Not sure if that's a good idea.

Reply:
I am going to leave this comment open for @niemeyer as he proposed the name. Happy to tweak it :)

Reply:
Changed to ```ErrorResponse```, resolving.